app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">pubgapp</string>
+    <string name="map_tab_title">Maps</string>
```

Comment:
delete these if we're not using them, but I think we should follow this pattern:

`${type}_${description}` i.e., `title_map`, `title_tab`